lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -89,6 +89,11 @@ def migrate(direction)
           when :down then announce "reverted (%.4fs)" % time.real; write
         end
 
+        begin
+          @@after_migrate.call(@buffer_output, name) if @@after_migrate
+        rescue => e
+          announce("Error in after_migrate hook: #{e}")
```

Comment:
Not sure for that. Maybe `Rails.logger.debug` is better.

Reply:
Looks like we use `announce` everywhere so why not, `say` is also a good option.
I don't think `Rails.logger.debug` is great because it would be hidden in the logs instead of visible in the output.